wrapper.go

```diff
@@ -32,10 +34,22 @@ const (
 func init() {
 	logger = log.New()
 	logger.Out = os.Stdout
-	logger.Formatter = &easy.Formatter{
-		TimestampFormat: "2006-01-02 15:04:05",
-		LogFormat:       "#LUMIGO# - %time% - %lvl% - %msg%\n",
+	logger.Formatter = &LogFormatter{}
+}
+
+//Log custom format
+type LogFormatter struct{}
+
+//Format details
+func (s *LogFormatter) Format(entry *log.Entry) ([]byte, error) {
+	timestamp := time.Now().Local().Format("2006-01-02 15:04:05")
+	msg := fmt.Sprintf("#LUMIGO#-%s-%s-message:%s ", timestamp, strings.ToUpper(entry.Level.String()), entry.Message)
+	for k, v := range entry.Data {
+		msg += fmt.Sprintf("%s:%v ", k, v)
+		entry.Data[k] = fmt.Sprintf("%v", v)
```

Comment:
Is this for json? Can't we just do something like `json.dumps`?

Reply:
sure